zb-constants.js

```diff
@@ -252,28 +252,40 @@ const STATUS = {
   INSUFFICIENT_SPACE: zclId.status('insufficientSpace').value,
 };
 
-// THERMOSTAT_MODE is used for the systemMode and runningMode attributes
+// THERMOSTAT_MODE is used for the systemMode attribute
 // for the hvacThermostat cluster.
-const THERMOSTAT_MODE = [
-  'Off',    // 0
-  'Auto',   // 1
+const THERMOSTAT_SYSTEM_MODE = [
+  'off',    // 0
+  'auto',   // 1
   null,     // 2 - not used in the spec
-  'Cool',   // 3
-  'Heat',   // 4
+  'cool',   // 3
+  'heat',   // 4
+  // There are other values, but these are the only ones we support
+  // now.
+];
+
+// THERMOSTAT_MODE is used for the runningMode attribute
+// for the hvacThermostat cluster.
+const THERMOSTAT_RUN_MODE = [
+  'off',      // 0
+  'auto',     // 1
```

Comment:
I find it weird that `auto` is actually a running mode.

Reply:
You're right - I think I was originally using the same table getting versus setting. For runningMode, the spec only defines values 0, 3, and 4